src/psyclone/algGen.py

```diff
@@ -1,23 +1,54 @@
 # -----------------------------------------------------------------------------
-# (c) The copyright relating to this work is owned jointly by the Crown,
-# Met Office and NERC 2014.
-# However, it has been created with the help of the GungHo Consortium,
-# whose members are identified at https://puma.nerc.ac.uk/trac/GungHo/wiki
+# BSD 3-Clause License
+#
+# Copyright (c) 2014-2019, Science and Technology Facilities Council.
```

Comment:
I thought the Met Office and NERC transferred copyright so that anything we wrote previously was owned by us (as long as it was BSD). As I started writing this code in 2014 I think it makes sense to reflect this. I could be wrong though :-).

Reply:
OK, that makes sense.